packages/workers-shared/asset-worker/src/compatibility-flags.ts

```diff
@@ -0,0 +1,55 @@
+import type { AssetConfig } from "../../utils/types";
+
+interface CompatibilityFlag {
+	enable: `assets_${string}`;
+	disable: `assets_${string}`;
+	onByDefaultAfter?: string;
+}
+
+export const SEC_FETCH_MODE_NAVIGATE_HEADER_PREFERS_ASSET_SERVING = {
+	enable: "assets_sec_fetch_mode_navigate_header_prefers_asset_serving",
+	disable: "assets_not_found_handling_prefers_asset_worker",
+	onByDefaultAfter: "2025-06-01", // TODO: Pick a date
```

Comment:
Flagging to myself. Once we're happy with the behavior, we can set this for April-ish or something.